.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,96 @@
+# For most projects, this workflow file will not need changing; you simply need
+# to commit it to your repository.
+#
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+#
+# ******** NOTE ********
+# We have attempted to detect the languages in your repository. Please check
+# the `language` matrix defined below to confirm you have the correct set of
+# supported CodeQL languages.
+#
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ develop, master, release/0.8.* ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ develop, master, release/0.8.* ]
+  schedule:
+    - cron: '0 0 * * 3'
```

Comment:
I can't cron, can you provide a comment on that this means? Every tuesday at midnight?

Reply:
very close: every Wednesday. not sure what timezone but probably UTC

Reply:
resolved in c0ab81674